lib/tools/adb-commands.js

```diff
@@ -532,6 +532,11 @@ methods.processExists = async function (processName) {
   }
 };
 
+methods.forwardList = async function () {
```

Comment:
I would prefer something like `getForwardList`. Semantically, `forwardPort` actually sets the port forwarding, so one would expect `forwardList` to set the list forwarding, which it does not.